pages/app/dashboards-detail/index.js

```diff
@@ -1,21 +1,58 @@
 import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import isEmpty from 'lodash/isEmpty';
 
 // actions
-import { getDashboard } from 'modules/dashboards/actions';
```

Comment:
@dancer2090 Your changes in this page have broken the dashboard preview pages, here's a url example where you can check that http://localhost:9000/data/dashboards/test-dashboard-is_highlighted

Reply:
If you want to check it with one of your dashboards, go to `MyRW` --> `Dashboards` and you will see the preview links in the cards.
![image](https://user-images.githubusercontent.com/545342/70150443-4f5d7e00-16aa-11ea-891e-6cf2bbace099.png)